### DISCUSSION TITLE
I need to create a model for the deformation of hydro-elastic contacts with temperature and vibration effects.

### DISCUSSION THREAD

**Discussion by Yasmeena424** [google-deepmind/mujoco#1429] [General]
I need to create a model for the deformation of hydro-elastic contacts with temperature and vibration effects.

**Comment by yuvaltassa** [comment]
This can probably be done in MuJoCo, but absent further context I recommend you take a look at [Drake](https://drake.mit.edu/doxygen_cxx/group__hydroelastic__user__guide.html).
